package.json

```diff
@@ -29,13 +29,12 @@
     "build:demo": "yarn workspace @evolvedbinary/prosemirror-lwdita-demo build",
     "build": "yarn workspace @evolvedbinary/prosemirror-lwdita-localization build && yarn workspace @evolvedbinary/prosemirror-lwdita build",
     "prepack": "yarn run build",
+    "lint": "eslint",
```

Comment:
I think lint should be setup per workspace just like the other targets please

Reply:
@adamretter I don't think that should be the case for linting, it should be project based here's my reasons:
1. We will have to add the dependency to all packages.
2. The eslint config file will be replicated 4 times.
3. I m not sure what effect it will have on the IDE linting as it uses eslint server, and it will be loading the config 4 times.
4. The existing setup uses root level linting
```json
"lint": "eslint packages/* --ext .ts",
"lint:fix": "eslint packages/* --ext .ts --fix",
```
5. I used the recommended way of adding eslint.